src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -390,6 +390,7 @@ private function addTableToSchema(Schema $schema): void
         $table->addColumn('headers', self::$useDeprecatedConstants ? Type::TEXT : Types::TEXT)
             ->setNotnull(true);
         $table->addColumn('queue_name', self::$useDeprecatedConstants ? Type::STRING : Types::STRING)
+            ->setLength(190) // https://github.com/symfony/symfony/issues/37116
```

Comment:
please replace the link with a real comment as we don't link github issues in code

Reply:
I see, I replaced the reference with a short explanation on why the the field has a length definition.